waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -60,22 +64,6 @@ genDb wasp _ =
   [ genPrismaSchema wasp
   ]
 
-preCleanup :: Wasp -> Path' Abs (Dir ProjectRootDir) -> CompileOptions -> IO ()
-preCleanup wasp projectRootDir _ = do
-  deleteGeneratedMigrationsDirIfRedundant wasp projectRootDir
-
-deleteGeneratedMigrationsDirIfRedundant :: Wasp -> Path' Abs (Dir ProjectRootDir) -> IO ()
```

Comment:
Removed this since we would always want to have a clean copy of migrations in generated project. So we just remove if present in project, and then copy Wasp migrations dir down if present.